Fun.Blazor/DIComponent.fs

```diff
@@ -50,6 +50,7 @@ type DIComponent<'T>() as this =
     let blazorLifecycle =
         { new IComponentHook with
             member _.OnAfterRender = afterRenderEvent.Publish
+            member _.OnInitialized = initialized.Publish
```

Comment:
I remember I added this before but for some reason I did not get triggerred so I removed it. I will check again after merging. Thanks!